docs/sdk/dev/initialize-and-run-workflows.md

```diff
@@ -38,11 +36,7 @@ spryker-sdk sdk:workflow:run {workflowName}
 You can start any workflow by providing its name in the `sdk:workflow:run` command.
 
 ---
-{% info_block infoBox "Identical workflows" %}
-
-Two identical top-level workflows can't run inside the same project.
-
-{% endinfo_block %}
+**Note:** Two identical top-level workflows can't run inside the same project.
```

Comment:
{% info_block warningBox "Identical workflow exclusion" %}

Two identical top-level workflows can't run inside the same project.

 {% endinfo_block %}